apps/cache-manager/src/app.ts

```diff
@@ -0,0 +1,42 @@
+import express from 'express'
+
+import {CONNECTION_STRING, TABLE_NAME} from './globals'
+import {AzureTableStorage, cacheExpired, getComponent, insertNewItem} from './utils'
+
+const app = express()
+app.use(express.json())
+
+app.post('/:port/prerender', async (req, res) => {
+  try {
+    let component
+
+    const remoteName = req.get('remote-name') || 'batman'
+
+    const client = await AzureTableStorage.connectTableClient(CONNECTION_STRING, TABLE_NAME)
+
+    const base64Body = Buffer.from(JSON.stringify(req.body)).toString('base64')
+
+    const tableRes: any = await AzureTableStorage.getTableItem(client, remoteName, base64Body)
+
+    if (tableRes === undefined) {
+      const response = await getComponent(req.body, req.params.port)
+      component = response.data
+      await insertNewItem(remoteName, base64Body, response.data, client)
```

Comment:
Do you need to await these calls? feels like we we don't need to

Reply:
yeah taken a few of them out, so were not waiting on the insert to complete before passing the response back